project/src/com/google/daggerqueryui/third-party/css/roboto-fonts.css

```diff
@@ -0,0 +1,252 @@
+/* cyrillic-ext */
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 300;
+  font-display: swap;
+  src: local('Roboto Light'), local('Roboto-Light'), url(https://fonts.gstatic.com/s/roboto/v20/KFOlCnqEu92Fr1MmSU5fCRc4AMP6lbBP.woff2) format('woff2');
```

Comment:
optional: Not sure if you want to make these offline as well if you're really concerned about internet availability. I wouldn't worry too much :stuck_out_tongue:

Reply:
I see, thanks for noticing this. Boostrap's [default font family](https://www.w3schools.com/bootstrap4/bootstrap_typography.asp#:~:text=Bootstrap%204%20uses%20a%20default,1rem%20(16px%20by%20default).) is "Helvetica Neue" and it looks pretty good too. If there are no special requirements for using `Roboto` (it's quite popular in Google) I would rather remove it rather than download a lot of extra files.

Reply:
Or we can leave everything as it is and use Roboto only if the Internet is available.

Reply:
Well it's just a prototype so I wouldn't worry too much about font requirements, let's just leave it aside :smiley: